tools/OWNERS

```diff
@@ -10,5 +10,9 @@
         {name: 'ampproject/wg-performance'},
       ],
     },
+    {
+      pattern: 'experiment-config.js',
+      owners: [{name: 'ampproject/reviewers-amphtml'}],
```

Comment:
One consideration: a review from `ampproject/reviewers-amphtml` is always required no matter what, so this could optionally be `name: '*'` or `name: 'ampproject/working-groups'`. Fine with this too though. Approving to unblock, up to you

Reply:
Follow-up: this team is a good idea for providing a team to the reviewer suggestion algorithm

Reply:
Oh I see. Good to know! I'll change this to `ampproject/working-groups`.

Reply:
This is done. If there are no more comments, I'll merge this after CI since this file tends to get a LOT of merge conflicts. =)